container_pipeline/workers/base.py

```diff
@@ -46,23 +48,36 @@ def export_logs(self, logs, destination):
 
     def run(self):
         """Run worker"""
+        current_job_name = None
+        previous_job_name = None
+        current_trig_time = datetime.time
+        previous_trig_time = datetime.time
+
         while True:
             try:
                 job_obj = None
+                previous_job_name = current_job_name
+                previous_trig_time = current_trig_time
                 job_obj = self.queue.get()
                 job = json.loads(job_obj.body)
+                current_job = get_job_name(job)
+                current_trig_time = datetime.time
                 debug_logs_file = os.path.join(
                     job['logs_dir'], settings.SERVICE_LOGFILE)
                 # Run dfh.clean() to clean log files if no error is
                 # encountered in post delivering build report mails to user
                 dfh = DynamicFileHandler(self.logger, debug_logs_file)
                 self.logger.info('Got job: {}'.format(job))
-                try:
-                    self.handle_job(job)
-                except Exception as e:
-                    self.logger.error(
-                        'Error in handling job: {}\nJob details: {}'.format(
-                            e, job), extra={'locals': locals()}, exc_info=True)
+                if((previous_job_name == current_job_name) and ((current_trig_time - previous_trig_time) < 1000)):
+                    self.logger.critical(
+                        'Current job is repeating frequently: Job details: {}'.format(job), extra={'locals': locals()}, exc_info=True)
```

Comment:
This should not be a critical log, rather ``info`` since we are able to handle this scenario.